Demo/Demo/Extensions/PaymentButtonEnums+Extension.swift

```diff
@@ -69,7 +69,27 @@ extension PaymentButtonFundingSource: CaseIterable {
 extension PayPalButton.Label: CaseIterable {
     
     public static var allCases: [PayPalButton.Label] {
-        [.none, .checkout, .buyNow, .payWith]
+        [
+            .addMoneyWith,
+            .bookWith,
+            .buyNowWith,
+            .buyWith,
+            .checkoutWith,
+            .continueWith,
+            .contributeWith,
+            .none,
```

Comment:
Nit - usually none comes first in the list of all cases

Reply:
Thanks, will make the update!